github_api/voting.py

```diff
@@ -118,15 +118,8 @@ def get_vote_weight(api, username):
     created = arrow.get(user["created_at"])
     age = (now - created).total_seconds()
     old_enough_to_vote = age >= settings.MIN_VOTER_AGE
-    age_weight = 1.0 if old_enough_to_vote else 0.0
+    weight = 1.0 if old_enough_to_vote else 0.0
```

Comment:
I'm always looking to reduce the number of conditionals, thus the cast, but I can see your point.

At the least, consider getting rid of the antipattern:

    weight = ...
    return weight

which has no documentary value here in a short function called `get_vote_weight`, leading to:

   return 1.0 if age >= settings.MIN_VOTER_AGE else 0.0